### PR TITLE
Added configurable daemon check time and fixed json parse issues. 

### PR DESCRIPTION
**Pull request by waynegemmell**

### CODE REVIEW

File: monit/config.sls [hunk 1]
```diff
@@ -8,8 +8,9 @@
     - makedirs: True
     - mode: '0700'
     - context:
-        config_includes: {{ monit.config_includes | json }}
-        http_access: {{ monit.http_access | json }}
+        config_includes: {{ monit.config_includes }}
+        http_access: {{ monit.http_access | tojson }}
```
Comment (myii) [inline monit/config.sls]: Unfortunately, we still have to stick to `| json` to support `2017.7` minions.

File: monit/config.sls [hunk 2]
```diff
@@ -19,7 +20,7 @@
     - template: jinja
     - makedirs: True
     - context:
-      mail_alert: {{ monit.mail_alert }}
+      mail_alert: {{ monit.mail_alert | tojson }}
```
Comment (myii) [inline monit/config.sls]: [Likewise](https://github.com/saltstack-formulas/monit-formula/pull/11/files#r271064097).

File: monit/config.sls [hunk 3]
```diff
@@ -29,4 +30,4 @@
     - source: salt://monit/files/modules
     - template: jinja
     - context:
-      modules: {{ monit.modules }}
+      modules: {{ monit.modules | tojson }}
```
Comment (myii) [inline monit/config.sls]: [Likewise](https://github.com/saltstack-formulas/monit-formula/pull/11/files#r271064097).

**Review by myii — CHANGES_REQUESTED** [review]
Just an eyeball review for now.  We can't use `| tojson` at the current time.

**Comment by waynegemmell** [comment]
I've made the changes from tojson to json. I hope it all pushes through properly.

File: monit/config.sls [hunk 4]
```diff
@@ -8,8 +8,9 @@
     - makedirs: True
     - mode: '0700'
     - context:
-        config_includes: {{ monit.config_includes | json }}
+        config_includes: {{ monit.config_includes }}
```
Comment (aboe76) [inline monit/config.sls]: why remove the ` | json` here?

Comment (waynegemmell) [reply monit/config.sls]: Oversight, or a mistake. You choose ;)
I've put it back now.

**Comment by waynegemmell** [comment]
This should be sorted now ?

File: monit/files/modules [hunk 5]
```diff
@@ -1,50 +1,17 @@
-# -*- coding: utf-8 -*-
-# vim: ft=jinja
```
Comment (myii) [inline monit/files/modules]: Is there a reason for removing these lines?

File: monit/files/modules [hunk 6]
```diff
@@ -1,50 +1,17 @@
...
-          with:
-            pidfile: /var/run/nginx.pid
-          config:
-            group: www
-            start: "/etc/init.d/nginx start"
-            stop: "/etc/init.d/nginx stop"
-          if: 
-            failed: 127.0.0.1 port 80 protocol http
-            action: restart
--#}
```
Comment (myii) [inline monit/files/modules]: Likewise, is there a reason for removing all of these comments from this file?

**Comment by myii** [comment]
@waynegemmell Thanks for fixing the `| json` issue.  I've just had a closer look and I've queried above about the necessity for all of the line removals -- I can't see it.  I prefer to leave merging for those who are more familiar with the formula or at least make use of it, so apologies for the delay.  In the meantime, I've also spotted that `iteritems` is being used.  Those can all be updated to `items` if you're willing to add that commit -- no problem if not, it can be dealt with in a subsequent PR for whoever is willing to do that.

File: monit/files/modules (hunk 5 above)
Comment (waynegemmell) [reply monit/files/modules]: No, my version is behind that one anyway. I've added these lines to my file.

File: monit/files/modules (hunk 6 above)
Comment (waynegemmell) [reply monit/files/modules]: Same story. I don't want to mess with that file as I don't understand it properly.

**Comment by waynegemmell** [comment]
Maybe in another PR. I'd like to get this through ASAP so I can use it live.

**Comment by myii** [comment]
@waynegemmell If you can replace the missing lines, I'll be in a position to merge this PR.

**Comment by myii** [comment]
@waynegemmell Thanks for fixing all of that, including the `iteritems`.  Sorry about the wait.  Merged!

**Pull request merged**